11/Zaubeerer/test_generators.py

```diff
@@ -0,0 +1,57 @@
+import generators
+from collections import Counter
+from itertools import islice
+
+FILE_PATTERN = '*.py'
+LINE_PATTERN = "import "
+
+def test_gen_files():
+    expected_files = ["test_generators.py", "generators.py"]
+
```

Comment:
I think you can get rid of the empty lines as these are short functions

Reply:
I have removed some empty lines, does it now suit your expectations? :)